src/constants/possibleResults.js

```diff
@@ -0,0 +1,37 @@
+export const possibleResults = {
+  synergy: {
+    id: 1,
+    name: "Low Risk & Synergy",
+    icon: "Inserte icono aqui",
+  },
+  noSynergy: {
+    id: 2,
+    name: "Low Risk & No Synergy",
+    icon: "icon",
+  },
+  decrease: {
+    id: 3,
+    name: "Low Risk & Decrease",
+    icon: "icon",
+  },
+  caution: {
+    id: 4,
+    name: "Caution",
+    icon: "icon",
+  },
+  unsafe: {
+    id: 5,
+    name: "Unsafe",
+    icon: "icon",
```

Comment:
Here the icon key value is going the be the `svg` component I mentioned above.

```
import CatuionSVG from '@assets/svgs/CautionSVG';

...
icon: CautionSVG,
...
```

Then you can consume them in the CombinationResult component.